tests/formats/xbm_test.zig

```diff
@@ -0,0 +1,9 @@
+const helpers = @import("../helpers.zig");
+const Image = @import("../../src/Image.zig");
+const ImageError = Image.Error;
+
+test "XBM: invalid file format" {
+    try helpers.expectError(helpers.testImageFromFile(helpers.fixtures_path ++ "xbm/bad_missing_dim.xbm"), ImageError.Unsupported);
+
+    try helpers.expectError(helpers.testImageFromFile(helpers.fixtures_path ++ "xbm/bad_missing_pixels.xbm"), Image.ReadError.InvalidData);
+}
```

Comment:
Where are the tests that actually test that what you read gives you the expected pixels?